bedrock/settings/base.py

```diff
@@ -734,6 +734,9 @@ def facebook_tab_url_lazy():
                             urlquote('utm_source=mozilla&utm_medium=Referral&'
                                      'utm_campaign=mozilla-org'))
 
+# Link to Firefox for iOS on Itunes
+ITUNES_FIREFOX_LINK = ('https://itunes.apple.com/nz/app/firefox-web-browser/id989804926')
```

Comment:
We already have a link in `base.py` to the appropriate app store link here: https://github.com/mozilla/bedrock/blob/master/bedrock/settings/base.py#L743